appserver/distributions/glassfish-common/src/main/resources/bin/debug-asadmin

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+#
+# Copyright (c) 2010, 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
I doubt this file has been created in 2010

Reply:
Well, it has been copied from a file that was created in 2010. So, I guess, the content is copyrighted from 201 onwards, right?

Reply:
I wouldn't be surprised if >90% of new files start as a copy of something :-D VCS sees this as a new file, so I'd use only `2020` here.

btw: why you just can't call existing `asadmin` with appropriate argument?

Reply:
> btw: why you just can't call existing asadmin with appropriate argument?

In `asadmin.sh` all arguments are passed through to the java process. Extracting a special argument then constructing a new array is not particularly pretty either, especially for maximum shell compatibility.